test/manual/templates/template_operations_test.sh

```
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Template Operations Tests - Filtering & Re-apply
+
+source "$(dirname "${BASH_SOURCE[0]}")/utils.sh"
+export RUN_NO_CLEANUP=true
+
+if [[ -f /tmp/basic_template_id.txt ]]; then
+  TEMPLATE_ID=$(< /tmp/basic_template_id.txt)
+  CREATED_HERE=0
+else
+  local raw payload
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)